styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
```

Comment:
Je bent de CSS reset van de margin en padding vergeten, dit is bijna altijd handig om te doen.